src/dataframe.ts

```diff
@@ -15,37 +23,81 @@ export interface DataFrame {
   numRows: number
   // Rows are 0-indexed, excludes the header, end is exclusive
   // if orderBy is provided, start and end are applied to the sorted rows
-  rows(args: RowsArgs): AsyncRow[]
+  // negative start and end are allowed
+  rows({ start, end, orderBy }: {start: number, end: number, orderBy?: string}): AsyncRow[]
```

Comment:
The types here should probably have whitespace between the curly braces. Wonder if there's a typescript eslint rule.

```suggestion
  rows({ start, end, orderBy }: { start: number, end: number, orderBy?: string }): AsyncRow[]
```

Reply:
done. Good question... I could look at it, maybe reopening https://github.com/hyparam/hyperparam-cli/issues/35, if you want me to take time for this

Reply:
Not a priority